core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -690,6 +705,62 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest with Dou
     insights.features.foreach(f => f.distributions shouldBe empty)
   }
 
+  it should
+    """include sensitive feature information
+      |even for sensitive features that are removed from output vector and output vector metadata""".stripMargin in {
+    // Copy metadata from above but add new feature that was removed in vectorizing to sensitive info
+    val f_notInMeta = Feature[Text]("f_notInMeta", false, null, Seq(), "test")
+    val newMeta = OpVectorMetadata(
```

Comment:
Can you clean up the creation of this object here? It would be easier to read if you construct the arguments individually in their own vals and then plug them in at the end so that you don't have to go to the definition of OpVectorMetadata to figure out what the args are supposed to be.